addons/point_of_sale/static/src/app/screens/product_screen/product_screen.xml

```diff
@@ -43,16 +43,56 @@
                     </div>
                 </div>
             </div>
-            <div class="rightpane overflow-auto d-flex flex-grow-1 flex-column bg-300 w-60" t-if="!ui.isSmall || pos.mobile_pane === 'right'">
-                <ProductsWidget />
+            <div class="rightpane d-flex flex-grow-1 flex-column bg-300 w-60" t-if="!ui.isSmall || pos.mobile_pane === 'right'">
+                <div t-attf-class="{{scrollDirection.down and ui.isSmall ? 'hide-control-top-bar' : 'd-flex'}} bg-view shadow-sm control-top-bar">
+                    <CategorySelector
+                        categories="getCategories()"
+                        onClick="(id) => this.pos.setSelectedCategoryId(id)" />
+                    <Input tModel="[pos, 'searchProductWord']"
+                        class="'ms-auto m-auto p-2'"
+                        isSmall="ui.isSmall"
+                        placeholder="'Search products...'"
+                        icon="{type: 'fa', value: 'fa-search'}"
+                        debounceMillis="100" />
+                </div>
+                <div t-if="productsToDisplay.length != 0 and pos.posHasValidProduct()" t-ref="products" class="overflow-y-auto flex-grow-1">
+                    <div t-attf-class="product-list {{this.pos.productListViewMode}} p-1">
+                        <ProductCard t-foreach="productsToDisplay" t-as="product" t-key="product.id"
+                            class="pos.productViewMode"
+                            name="product.display_name"
+                            productId="product.id"
+                            price="product.getFormattedUnitPrice()"
+                            imageUrl="product.getImageUrl()"
+                            onClick="() =>this.pos.addProductToCurrentOrder(product)"
+                            productInfo="true"
+                            onProductInfoClick="() => this.onProductInfoClick(product)"
+                        />
+                    </div>
+                    <div t-if="searchWord" class="search-more-button d-flex justify-content-center m-2">
+                        <button class="btn btn-primary btn-lg" t-on-click="onPressEnterKey">Search more</button>
+                    </div>
```

Comment:
move this outside the t-if div such that it appears even when there are no products